lib/w_common_browser.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Workiva Inc.
```

Comment:
Personally I don't think we even need this entry point. I think it's better practice for consumers to import the piece that they need (`disposable_browser.dart`) instead of everything, but I'll defer to the masses if need be.